packages/blade/src/components/Button/BaseButton/getBaseButtonStyles.ts

```diff
@@ -2,18 +2,11 @@ import type { CSSObject } from 'styled-components';
 import type { StyledBaseButtonProps } from './StyledBaseButton';
 
 const getBaseButtonStyles = ({
-  activeBorderColor,
```

Comment:
why removed this? because this is not needed on RN?

Reply:
Don't need it in common styles, the active border color is handled on the native component `StyledBaseButton.native.tsx`